libcontainer/container_linux.go

```diff
@@ -596,6 +638,11 @@ func (c *Container) newInitProcess(p *Process, cmd *exec.Cmd, messageSockPair, l
 		return nil, err
 	}
 
+	// Send idmap sources.
```

Comment:
Micro-nit (but in case there's other changes to make); I'd remove this comment, as it's not adding much value (the function is named `c.sendIdmapSources`, so it's describing the code, not the intent)

Reply:
Fixed, thanks!